content/services/planning.md

```diff
@@ -157,6 +168,18 @@ summary: |
   with interactive wireframes,
   prioritized estimates,
   and a robust plan for development and launch.
+
+  Our [certified object-oriented user experience strategist]
+  will guide you through a process to define the concepts 
+  or "objects" that make up your web app, 
```

Comment:
I wonder if this will always be true ... Are there cases when you (@SondraE) aren't the one working with the client on this early phase? I think for GSM I started the data map (or some project, can't remember) but then you came in to clean it up. I realize you have done the majority, but is it a problem if an uncertified UX person would do this on a project in the future based on availability? I am also perfectly happy with how it is worded and if there is a project in the future that doesn't follow this process, we note it on an as needed basis (like a lot of our smaller projects that don't go through the "usual" process.)

Reply:
Good point. How about this? "Using an object-oriented user experience strategy, we'll guide you through a process to define the concepts or "objects" that..."

Reply:
perfect @SondraE